test/tableau/extensions/sitemap_extension_test.exs

```diff
@@ -0,0 +1,76 @@
+defmodule Tableau.SitemapExtensionTest do
+  use ExUnit.Case, async: true
+
+  alias Tableau.SitemapExtension
+
+  test "use _site as default out dir" do
+    token = %{
+      site: %{
+        config: %{
+          url: "http://example.com",
+          out_dir: "_site"
```

Comment:
You want to set this to the tmp dir as well

Reply:
Thank you